src/gui/modulelisteditor_funcs.cpp

```diff
@@ -146,15 +143,25 @@ void ModuleListEditor::disableSensitiveControls()
 {
     ui_.AvailableModulesTree->setEnabled(false);
     chartWidget_->disableSensitiveControls();
-    ui_.ControlsWidget->disableSensitiveControls();
+    for (int n = 0; n < ui_.ModuleControlsStack->count(); ++n)
+    {
+        ModuleControlWidget *mcw = dynamic_cast<ModuleControlWidget *>(ui_.ModuleControlsStack->widget(n));
+        if (mcw)
+            mcw->disableSensitiveControls();
+    }
```

Comment:
```suggestion
    for (ModuleControlWidget *mcw : ui_.ModuleControlsStack)
    {
        if (mcw)
            mcw->disableSensitiveControls();
    }
```

Same caveat as above

Reply:
Alas, also a "No". `QStackedWidget` does have a `children()` function that returns an iterable `QObjectList`, but attempting to cast these up to the necessary `ModuleControlWidget`s gives a hard crash.